### ISSUE TITLE
Request for Code Assistance on Training and Validation Loss and AUC Plot 

### ISSUE THREAD

**Issue by Yasich** [unnir/DeepTLF#2]
Hello. I hope this message finds you well.
Thank you very much for developing this method and sharing it.
I need Loss Diagram for Training and Validation set, AUC Plot for Validation and Training set (AUC vs epoch). Could you please assist me in writing the code for these plots? 

Thank you in advance for your time and consideration.
@unnir
